ui/src/components/Main.svelte

```diff
@@ -32,6 +33,12 @@
     });
   });
 
+  window.addEventListener('message', (event) => {
```

Comment:
Use this insted https://github.com/Project-Sloth/ps-dispatch/blob/3988e28ec86df76b9ff51e465a7c1a2e9145e52b/ui/src/providers/AlwaysListener.svelte#L48

Reply:
The reciveNUI should look like this, remember to import the new RESPOND_KEYBIND store

	ReceiveNUI('setupUI', (data: any) => {
		PLAYER.set(data.player)
		Locale.set(data.locales)
	        RESPOND_KEYBIND .set(data.locales)
	});

Reply:
done